### PR TITLE
f2c: ratio android

### PR DESCRIPTION
**Pull request by skylarbarrera**
fixes APP-489
## What changed (plus any additional context for devs)
plaid package was getting loaded twice and causing issues: https://github.com/plaid/react-native-plaid-link-sdk/pull/306#issuecomment-1458940785

fixed an issue with the isAuthenticated logic that didnt handle android devices properly as well as a edge case where the user has only imported a watch wallet and wants to fund that wallet (no pin set up yet)

## Screen recordings / screenshots
https://cloud.skylarbarrera.com/Screen-Recording-2023-03-15-16-38-43.mp4

## What to test

lfg

### CODE REVIEW

File: src/utils/authentication.ts [hunk 1]
```diff
@@ -19,14 +25,16 @@ async function maybeSaveFakeAuthKey() {
 }
 
 export async function isAuthenticated() {
-  if (IS_IOS) {
+  const hasBiometricsEnabled = await getSupportedBiometryType();
+  if (hasBiometricsEnabled) {
```
Comment (estrattonbailey) [inline src/utils/authentication.ts:29]: Might this be true for Android also?

**Comment by skylarbarrera** [comment]
android sheet tweaks: https://cloud.skylarbarrera.com/Screen-Recording-2023-03-15-17-15-11.mp4

File: src/utils/authentication.ts (hunk 1 above)
Comment (skylarbarrera) [reply src/utils/authentication.ts:29]: yes it should be 

only want to fallback to pin on android

**Review by estrattonbailey — APPROVED** [review]

**Comment by estrattonbailey** [comment]
Love it

**Review by ibrahimtaveras00 — APPROVED** [review]
Android is now working (both PIN & Fingerprint) and iOS still working too 👍🏽

**Review by jkadamczyk — APPROVED** [review]
Looks great in code and on device

**Pull request merged**